src/OSFramework/Maps/FileLayer/IFileLayer.ts

```diff
@@ -7,8 +7,7 @@ namespace OSFramework.Maps.FileLayer {
 		fileLayerEvents: Event.FileLayer.FileLayersEventsManager;
 		isReady: boolean;
 		map: OSMap.IMap; //IMap
-		// eslint-disable-next-line @typescript-eslint/no-explicit-any
-		provider: any;
+		provider: unknown;
```

Comment:
@rugoncalves can't we have here something `GoogleMaps | Leaflet`

Reply:
In this "layer", we're not having any reference to any of the providers... I thought about creating an interface that would represent the provider, but stumbled upon considerable side effects.